web/app/js/components/util/EdgesUtils.jsx

```diff
@@ -13,15 +12,15 @@ export const processEdges = (rawEdges, resourceName) => {
       return;
     }
     // check if any of the returned edges match the current resourceName
-    if (_startsWith(edge.src.name, resourceName)) {
+    if (edge.src.name === resourceName) {
```

Comment:
any idea why we were using `_startsWith` in the first place?

Reply:
Good question, thanks for asking! I believe that as I began writing `EdgesUtils` for #2965, I originally assigned each row in `rawEdges` a `key` and was using `_startsWith` to check if that `key` started with the given `resourceName`. As I simplified the logic, I no longer filtered by `key`, but I didn’t catch the error. This happened before I put up the PR so the original record is lost to history.